pkgs/by-name/pa/passhole/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+}:
+let
+  pname = "passhole";
+  version = "1.10.0";
+
+  pykeepass-cache = python3.pkgs.buildPythonPackage {
+    pname = "pykeepass-cache";
+    version = "2.0.3";
+
+    src = fetchPypi {
+      pname = "pykeepass-cache";
+      version = "2.0.3";
+      sha256 = "sha256-fzb+qC8dACPr+V31DV50ElHzIePdXMX6TtepTY6fYeg=";
+    };
+
+    dependencies = with python3.pkgs; [
+      pykeepass
+      rpyc
+      python-daemon
+    ];
+  };
+in
+python3.pkgs.buildPythonPackage rec {
+  inherit pname version;
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3pNxhqmkw8tO0wVBuZnua64HJrEpyeNSAKiAMsM4oVY=";
+  };
+
+  dependencies =
+    with python3.pkgs;
+    [
+      pynput
+      pykeepass
+      colorama
+      future
+      pyotp
+      qrcode
+    ]
+    ++ [ pykeepass-cache ];
+
+  doCheck = true;
+  checkPhase = ''
+    ${python3}/bin/python test/tests.py
+  '';
+
+  meta = with lib; {
```

Comment:
Why though? I ran a quick `grep -rI "meta = with lib;.*" | wc -l` and I got 22961 results. So, I think the syntax is already pretty favored in the codebase. Is there a specific reason for you suggesting otherwise?

Reply:
Changed to not use lib in 6658d18c3762fc3fea7c2a7543771a89699e6d99.